adam-core/src/main/scala/org/bdgenomics/adam/rdd/ADAMContext.scala

```diff
@@ -1156,18 +1157,20 @@ class ADAMContext(@transient val sc: SparkContext) extends Serializable with Log
    * @return Returns a FeatureRDD.
    */
   def loadGff3(filePath: String,
+               cache: Boolean = true,
```

Comment:
No prob; we all have our thick days. What you suggested (passing the StorageLevel) is what I was thinking.